modules/jaeger-removing-instance.adoc

```diff
@@ -0,0 +1,27 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-removing.adoc
+////
+
+[id="jaeger-removing_{context}"]
+= Removing a Jaeger instance using the web console
+
+====
+NOTE: When deleting an instance that uses the in-memory storage, all data will be permanently lost. Data stored in a persistent storage (such as Elasticsearch) will not be deleted when a Jaeger instance is removed.
+====
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. If necessary, select the name of the project where you installed the Operators from the Project menu, for example, `jaeger-system`.
```

Comment:
Can if necessary be more specific? Like If you have already installed operators, select the name of the project from the *Project* menu, for example, `jaeger-system`.